constants/projects.json

```diff
@@ -0,0 +1,21 @@
+{
+    "data": [
+        {
+            "img": "https://example.com",
+            "title": "Morrent",
+            "desc": "A Car Rental Application",
+            "frontEnd": ["ReactJS","NextJS"],
+            "backEnd": ["NodeJS", "MongoDB"],
+            "color": "#416CEA"
+        },
+        {
+            "img": "https://example.com",
+            "title": "JobIT",
+            "desc": "A Job Finding Application",
+            "frontEnd": ["ReactJS","NextJS"],
+            "backEnd": ["NodeJS", "MongoDB"],
+            "color": "#0BAB7C"
+        }
```

Comment:
Excellent blocking out data. Now you already know what you'll need in sanity when the integration comes!